HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -239,6 +239,15 @@ def customizeHLTfor38761(process):
 
      return process
 
+def customizeHLTfor40264(process):
+    for producer in producers_by_type(process, "SiPixelPhase1MonitorVertexSoA"):
+        producer._TypedParameterizable__type = "SiPixelMonitorVertexSoA"
```

Comment:
this feels hacky... Is there any better option to rename on the fly?

Reply:
Thanks Marino @missirol for the suggestion. I have implemented it in https://github.com/cms-sw/cmssw/pull/40264/commits/dfe16cb1cedd2d9c04d51688ebf2e8cbcc0df761.
I expect that - eventually - after the alpaka migration is completed, this DQM plugins could be removed from the HLT menu itself and then run on DQM machines as part of a standalone client (as for the other subsystems). But that's likely for after January, so let's go with the way you propose.

Reply:
Thanks, sounds good.

Fwiw, playing around with the python customisation, this is arguably less hacky (but I'm not asking to make further changes):
```py
process.hltXYZ = cms.EDProducer('NewName', **process.hltXYZ.parameters_())
```

Reply:
Does the last point in https://its.cern.ch/jira/browse/CMSHLT-2603 imply we can go ahead an remove

https://github.com/cms-sw/cmssw/blob/689324234dd4e6a633578b1c39429371f08297dc/DQM/SiPixelHeterogeneous/plugins/SiPixelCompareVertexSoA.cc#L187-L188

and

https://github.com/cms-sw/cmssw/blob/a7263f73bf9a948a9e2fb997a4ccb32c3a548ba4/DQM/SiPixelHeterogeneous/plugins/SiPixelMonitorVertexSoA.cc#L133-L134

or will it be done together with an update of the HLT menu?

Reply:
The HLT menus first need to be updated in the release (will be done tomorrow). The duplicate plugins can then removed (I would prefer that's done in a separate PR).